integrations/lambda/demo.ipynb

```diff
@@ -1098,12 +1099,54 @@
     "\n",
     "# Attach Policy to the IAM Role\n",
     "policy_arn = \"arn:aws:iam::aws:policy/AmazonTimestreamFullAccess\"\n",
-    "iam_client.attach_role_policy(\n",
-    "    RoleName=role_name,\n",
-    "    PolicyArn=policy_arn\n",
-    ")\n",
     "\n",
-    "print(f\"Attached Timestream write policy to {role_name}\")\n",
+    "timestream_write_policy = {\n",
+    "    \"Version\": \"2012-10-17\",\n",
+    "    \"Statement\": [\n",
+    "        {\n",
+    "            \"Effect\": \"Allow\",\n",
+    "            \"Action\": [\n",
+    "                \"timestream:WriteRecords\",\n",
+    "                \"timestream:Select\",\n",
+    "                \"timestream:DescribeTable\",\n",
+    "\t\t\t\t\"timestream:CreateTable\"\n",
```

Comment:
Formatting

Reply:
Fixed. This must have happened because I copied and pasted the code from one of our other connectors.